tsconfig.app.json

```diff
@@ -3,7 +3,10 @@
   "extends": "./tsconfig.json",
   "compilerOptions": {
     "outDir": "./out-tsc/app",
-    "types": []
+    "types": [],
+    "target": "es2018",
+    "module": "es2020",
+    "lib": ["es2020", "dom"],
```

Comment:
Why is this required?

Reply:
The change was to prevent from raising errors below for `flatMap`:
```
Property 'flatMap' does not exist on type 'Person[]'. Do you need to change your target library? Try changing the 'lib' compiler option to 'es2019' or later
```

Since we will get organizations from challenge object soon, I will just remove organization part (where I used `flatMap`) and change `tsconfig.app.json` this back to what it was.